index.src.html

```diff
@@ -564,15 +606,47 @@ <h4 id="algorithm-store" algorithm>Store a `Credential`</h4>
 
     4.  Return a {{Promise}} which resolves with the result of executing |credential|'s
         [=interface object=]'s {{Credential/[[Store]](credential)}} internal method on |credential|.
-  </ul>
+  </ol>
+
+  <h4 id="algorithm-create" algorithm>Create a `Credential`</h4>
```

Comment:
This isn't quite the right name, since for public-key credentials we want to create the private key. That is, it's creating the credential source/generator/capability and not just the credential.